doc/fundamentals/distributed_computing.md

```diff
@@ -0,0 +1,115 @@
+# Distributed Computing
+
+Distributing computation across multiple workers is a fundamental part of processing any big data workload. There are two main questions to think about when trying to distribute computation:
+
+1) How many workers to spread computation across?
+
+2) How to define which workers are responsible for which data?
+
+## Pachyderm Workers
+
+Before we dive into the above questions, there are a few details you should understand about Pachyderm workers. 
+
+Pachyderm workers are spun up when you create the pipeline and are left running in the cluster waiting for new jobs (data) to be available for processing (committed). This saves having to recreate and schedule the worker pod for every new job.
+
+Every worker for a given pipeline is an identical pod running the Docker image you specified in the [pipeline spec](../reference/pipline_spec.html). Your analysis code does not need do anything special to run in a distributed fashion. Instead, Pachyderm will spread out the data that needs to be processed across the various workers and make that data available for your code. 
+
+## Controlling the Number of Workers (Parallelism)
+
+The number of workers that are used for a given pipeline is controlled by the `parallelism_spec` defined in the pipeline specification.
+
+```
+  "parallelism_spec": {
+    "strategy": "CONSTANT"|"COEFFICIENT"
+    "constant": int        // if strategy == CONSTANT
+    "coefficient": double  // if strategy == COEFFICIENT
+```
+Pachyderm has two parallelism strategies: `CONSTANT` and `COEFFICIENT`.
+
+If you use the `CONSTANT` strategy, Pachyderm will start the number of workers that you specify. To use this strategy, set the field strategy to `CONSTANT`, and set the field `constant` to an integer value (e.g. `"constant":10` to use 10 workers).
+
+If you use the `COEFFICIENT` strategy, Pachyderm will start a number of workers that is a multiple of your Kubernetes cluster’s size. To use this strategy, set the field `coefficient` to a double. For example, if your Kubernetes cluster has 10 nodes, and you set `"coefficient": 0.5`, Pachyderm will start five workers. If you set it to 2.0, Pachyderm will start 20 workers (two per Kubernetes node).
+
+**NOTE:** The parallelism_spec is optional and will default to `“coefficient": 1”`, which means that it'll spawn one worker per Kubernetes node for this pipeline. 
+
+## Sharding Data Across Workers (Glob Patterns)
```

Comment:
This seems to be the first mention of "sharding."  It looks like we are wanting this to be connected with question number (2) above.  Maybe mention sharding up there or rephrase this to more specifically connect it with the above question.  Also, you might consider: 

1. internally linking each of the two questions above with the respective section below.
2. actually adding the (1) and (2) numbers to the section titles.